pkg/utils/names_test.go

```diff
@@ -209,3 +209,68 @@ func TestValidateLabelValue(t *testing.T) {
 		}
 	}
 }
+
+// TestValidateNamespace tests that only correct Kubernetes namespace names are accepted
+func TestValidateNamespace(t *testing.T) {
+	cases := []struct {
+		In    string
+		Valid bool
+	}{
+		// Valid namespaces
+		{"default", true},
+		{"kube-system", true},
+		{"my-namespace", true},
+		{"myapp", true},
+		{"my-app-123", true},
+		{"prod", true},
+		{"test-123", true},
+		{"a", true},
+		{"a-b", true},
+		{"abc-123-xyz", true},
+		{"123app", true},     // DNS-1123 allows starting with number
+		{"123invalid", true}, // DNS-1123 allows starting with number
+		{"1", true},          // single number is valid
```

Comment:
in [K8s docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) I see: As a result, all namespace names must be valid [RFC 1123 DNS labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names).

This further specifies: 
```
contain at most 63 characters
contain only lowercase alphanumeric characters or '-'
start with an alphabetic character
end with an alphanumeric character
```

so `1` cant be a valid namespace?